src/demo/app/examples/append-to-shadow-root-example/append-to-shadow-root-example.component.html

```diff
@@ -0,0 +1,14 @@
+<p>By default <b>appendTo</b> is used to append dropdown into document scope.
+    However when working with ShadowDom isolation - that doesn't work well.</p>
+<p>ShadowDom has shadowRoot inside which it makes sense to append dropdown - to apply ng-select styles injected into shadow dom.</p>
+<p>Using <b>appendToShadowRoot</b> it's possible to set shadowRoot parent to append dropdown in. 
+    Usually that will be web component name or just name of element where ShadowDom isolation is enabled</p>
+<sample-shadow-component>
+    <ng-select [items]="people | async"
+               appendToShadowRoot="sample-shadow-component"
```

Comment:
What cases might you not just be referencing the parent (or an ancestor?). I'm wondering if the selector could be automatically determined.